packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/About.5.doc.mdx

```diff
@@ -7,13 +7,76 @@ import FlowsheetDataGridPropsTable from 'terra-data-grid/src/FlowsheetDataGrid?d
+|---|---|
+|Up Arrow       | Moves focus one cell up. If focus is on the top cell in the column, focus does not move.|
+|Down Arrow     | Moves focus one cell down. If focus is on the bottom cell in the column, focus does not move.|
+|Right Arrow    | Moves focus one cell to the right. If focus is on the right-most cell in the row, focus does not move.|
+|Left Arrow     | Moves focus one cell to the left. If focus is on the left-most cell in the row, focus does not move.|
+|Home           | Moves focus to the first cell in the row that contains focus.|
+|End            | Moves focus to the last cell in the row that contains focus.|
+|Control + Home (Windows) <br/> or <br/> Ctrl + Command + Left Arrow (Mac) | Moves focus to the first cell in the first non header row.|
+|Control + End (Windows) <br/> or <br/> Ctrl + Command + Right Arrow (Mac) | Moves focus to the last cell in the last row.|
+|Tab            | Moves focus out of the grid to the next focusable element in the tab sequence. If grid navigation is disabled thereby trapping focusing within a single grid cell, tab moves focus circularly amongst the focusable elements within the grid cell where focus is trapped.|
+|Shift + Tab    | Moves focus out of the grid to the previous focusable element in the tab sequence. If grid navigation is disabled thereby trapping focusing within a single grid cell, tab moves focus circularly amongst the focusable elements within the grid cell where focus is trapped.|
+|Space| Selects the focused cell when row selection mode is off or Selects the focused row when row selection mode is on.|
+|Enter| Disables grid navigation and allows navigation within the focused cell. If the cell contains focusable content, focus will move to the first focusable element in the cell. Pressing Escape will restore grid navigation.|
+|Escape| Removes all cell selections.<br/> When grid navigation is off, escape restores the grid navigation.|
+|Shift + Up|Extends cell selection one row up. The selection does not extend beyond the first non-header row.|
+|Shift + Down|Extends cell selection one row down. The selection does not extend beyond the last row.|
+|Shift + Left|Extends cell selection one column to the left. The selection does not extend to the row header column.|
+|Shift + Down|Extends cell selection one column to the right. The selection does not extend beyond the last column.|
+|Shift + Space|Selects the entire range of rows and columns between the starting and selected cells.|
+
+### Assistive Technology Support
+#### WAI ARIA Roles and States
+* [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)
+  * **FlowsheetDataGrid**: When the [ariaLabel](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#grid-props-table) property is specified, the screen reader uses the aria-label value as the [accessible name](https://w3c.github.io/accname/#dfn-accessible-name) for the grid.
```

Comment:
it looks like the link is broken: /components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#grid-props-table.